shared/github.ts

```diff
@@ -35,3 +35,19 @@ export interface GithubPullRequestEvent {
   pull_request: PullsGetResponse
   repository: ReposGetResponse;
 };
+
+/** Pushes a status to github for a commmit. */
+export async function setStatusOnGithub(
+  org: string, repo: string, sha: string, state: 'error' | 'failure' | 'pending' | 'success', 
+  description: string = '', target_url: string = '', context: string = 'branch-manager') {
+  const url = `https://api.github.com/repos/${org}/${repo}/statuses/${sha}`;
+  const config = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'Authorization': 'token <insert token here>',
```

Comment:
Is that intended to be left here?

Reply:
I'll add a TODO for loading the token at runtime.